gcsio/src/main/java/com/google/cloud/hadoop/gcsio/cooplock/CoopLockOperationDao.java

```diff
@@ -361,6 +357,17 @@ private void writeOperation(
         MILLISECONDS);
   }
 
+  private static String encodeRenameLogRecord(Map.Entry<FileInfo, URI> record) {
```

Comment:
Any reason these are not methods on RenameOperationLogRecord?

Reply:
RenameOperationLogRecord is a POJO. Inlined serialization because it's just a single method call.